official/resnet/keras/resnet_model.py

```diff
@@ -230,8 +230,7 @@ def resnet50(num_classes, dtype='float32', batch_size=None):
   x = identity_block(x, 3, [512, 512, 2048], stage=5, block='b')
   x = identity_block(x, 3, [512, 512, 2048], stage=5, block='c')
 
-  rm_axes = [1, 2] if backend.image_data_format() == 'channels_last' else [2, 3]
```

Comment:
why this change? how is this out of sync? 
cc @haoyuz

Reply:
Thanks Priya for bring this to my attention! Please don't change the ResNet model code. This PR was just merged earlier today for performance improvement.

Reply:
OK. It is hard to revert from a squashed changes. I will manually revert to the original.